src/common/lib/types/message.ts

```diff
@@ -9,6 +9,32 @@ import * as API from '../../../../ably';
 import { IUntypedCryptoStatic } from 'common/types/ICryptoStatic';
 import { MsgPack } from 'common/types/msgpack';
 
+const MessageActionNumberMap: Record<number, API.MessageAction> = {
+  0: 'MESSAGE_UNSET',
+  1: 'MESSAGE_CREATE',
+  2: 'MESSAGE_UPDATE',
+  3: 'MESSAGE_DELETE',
+  4: 'ANNOTATION_CREATE',
+  5: 'ANNOTATION_DELETE',
+};
```

Comment:
an object whose keys are a sequential index and whose values are strings is just an array of strings in js

Reply:
Will update this to just an array